auth/store_test.go

```diff
@@ -38,6 +38,33 @@ func dummyIndexWaiter(index uint64) <-chan struct{} {
 	return ch
 }
 
+// TestNewAuthStoreRevision ensures newly auth store
```

Comment:
there probably should be an integration test that restarts the etcd server and checks the hash stays the same / fails without this patch